info/README.md

```diff
@@ -0,0 +1,7 @@
+UPDATE MAIN README
+
+* To update the codestyle text in the main README, you **NEED TO CHANGE** the content of the file `guide-chapter-<N>.md`, contained in `info/guide`, the corresponding section of the rules that you changed / added.
```

Comment:
Better include the hyperlink to the README:

```markdown
[README.md](../README.md)
```

Reply:
Done [78342c5](https://github.com/saveourtool/diktat/pull/1384/commits/78342c53dd92a42802f781323a6cc10b03605118)

Please check it again